### PR TITLE
Update plugin-xlrelease-plugin.yml

### PR DESCRIPTION
**Pull request by saraswat2385**

### CODE REVIEW

**Comment by daniel-beck** [comment]
Would like to see confirmation by @byaminov that this is okay.

**Comment by byaminov** [comment]
@daniel-beck this is OK, thanks in advance! @saraswat2385 can you please add me there as well, for clarity (`byaminov`)?

File: permissions/plugin-xlrelease-plugin.yml [hunk 1]
```diff
@@ -4,4 +4,7 @@ paths:
 - "com/xebialabs/ci/xlrelease-plugin"
 developers:
 - "jdewinne"
-- "mbigorne"
\ No newline at end of file
+- "mbigorne"
+- "msaraswat"
```
Comment (batmat) [inline permissions/plugin-xlrelease-plugin.yml]: does not exist https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=msaraswat

**Comment by daniel-beck** [comment]
@batmat Thanks, missed that. @saraswat2385 Please see the documentation at https://github.com/jenkins-infra/repository-permissions-updater#requesting-permissions for more information.

While the current (invalid) configuration is merged, the permissions will not be updated until fixed.

**Pull request merged**